tests/lapi/math_tointeger_test.lua

```diff
@@ -0,0 +1,36 @@
+--[[
+SPDX-License-Identifier: ISC
+Copyright (c) 2023-2025, Sergey Bronnikov.
+
+18 – The Mathematical Library
+https://www.lua.org/pil/18.html
+
+6.7 – Mathematical Functions
+https://www.lua.org/manual/5.3/manual.html#6.7
+
+Synopsis: math.tointeger(x)
+]]
+
+local luzer = require("luzer")
+local test_lib = require("lib")
+
+-- The function `math.tointeger()` is available since Lua 5.3.
+if not test_lib.lua_current_version_ge_than(5, 3) then
```

Comment:
Minor: I suggest providing the funciton `lua_current_version_lt()`, which returns `not test_lib.lua_current_version_ge_than(5, 3)`. It helps to read these checks, I suppose. Feel free to ignore.